features/step_definitions/user_steps.rb

```diff
@@ -107,8 +107,8 @@
 end
 
 Given /^many posts from alice for bob$/ do
-  alice = FactoryGirl.create(:user_with_aspect, :username => 'alice', :email => 'alice@alice.alice', :password => 'password', :getting_started => false)
-  bob = FactoryGirl.create(:user_with_aspect, :username => 'bob', :email => 'bob@bob.bob', :password => 'password', :getting_started => false)
+  alice = FactoryBot.create(:user_with_aspect, :username => 'alice', :email => 'alice@alice.alice', :password => 'password', :getting_started => false)
```

Comment:
- Style/HashSyntax: Use the new Ruby 1.9 hash syntax.
- Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.
- Layout/LineLength: Line is too long. [151/120]